Views/InputsView.swift

```diff
@@ -321,23 +323,111 @@ class InputsView: UIView, UITableViewDataSource, UITableViewDelegate, UISearchBa
     return true
   }
 
-  private func loadSampleInputs() {
-    createSeed(registered: true, variety: "Variété 1", specie: "Espèce 1")
-    createSeed(registered: true, variety: "Variété 2", specie: "Espèce 2")
+  private func loadRegisteredInputs() {
+    createSeed(variety: "Variété 1", specie: "Espèce 1")
+    createSeed(variety: "Variété 2", specie: "Espèce 2")
 
-    createPhyto(registered: true, name: "Nom 1", firmName: "Marque 1", maaID: 1000, reentryDelay: 1)
-    createPhyto(registered: true, name: "Nom 2", firmName: "Marque 2", maaID: 2000, reentryDelay: 2)
-    createPhyto(registered: true, name: "Nom 3", firmName: "Marque 3", maaID: 3000, reentryDelay: 3)
+    let assets = openAssets()
+    let decoder = JSONDecoder()
 
-    createFertilizer(registered: true, name: "Nom 1", nature: "Nature 1")
-    createFertilizer(registered: true, name: "Nom 2", nature: "Nature 2")
-    createFertilizer(registered: true, name: "Nom 3", nature: "Nature 3")
-    createFertilizer(registered: true, name: "Nom 4", nature: "Nature 4")
+    do {
+      //let registeredSeeds = try decoder.decode([RegisteredSeed].self, from: assets[0].data)
```

Comment:
Why aren't the Seeds treated same as the Phyto products or Fertilizers?

Reply:
We wanted with Jonathan to both learn how to parse json, and seeds is one of its part. So I didn't want to add the same treatment for seeds because he worked on it. Don't worry the treatment will still be the same for each of the 3 input types